src/Silicon.cpp

```diff
@@ -568,7 +568,7 @@ namespace galsim {
 
             // Now we add in a displacement due to diffusion
             if (_diffStep != 0.) {
-                double diffStep = std::max(0.0, diffStep_pixel_z * (zconv - 10.0));
+	      double diffStep = std::max(0.0, diffStep_pixel_z * std::sqrt(zconv * _sensorThickness));
```

Comment:
I guess the `- 10.0` bit was a hack before to help the results match up better?  Anyway, the change means that the `std::max(0, ...)` is not necessary anymore.  The second value can no longer be negative.

Reply:
(1) Yes, the -10.0 was a hack I removed.  I've now removed the std::max.
(2) When I tried the rebase, I got the error below.  I couldn't figure out how to fix it.  Any advice?
(3) I've fixed most of the test_sensor.py errors, but I'm still working through a few of them.

dhcp42067:GalSim cslage$ git rebase --onto master releases/2.0
fatal: Needed a single revision
Does not point to a valid commit: master

Reply:
OK.  Don't worry about the rebase.  Once we're ready to go, I can do it before we merge.

Reply:
BTW, I think I figured out what the problem probably is.  You probably don't have a local copy of the master branch.  So you can either do
```
git rebase --onto origin/master releases/2.0
```
or
```
git checkout master  # to get your own copy of master
git checkout diffusion_tweak
git rebase --onto master releases/2.0
```
or indeed don't bother with either.  I don't mind resolving this after you get the tests working if you don't want to try to deal with git rebasing.  :)